.github/workflows/rust.yml

```diff
@@ -135,17 +136,23 @@ jobs:
       - name: fetch params
         run: forest-cli fetch-params --keys
       - name: download snapshot
-        run: forest-cli --chain calibnet snapshot fetch --aria2 -s /tmp/snapshots/
+        run: forest-cli --chain calibnet snapshot fetch --aria2 -s $SNAPSHOT_DIRECTORY
       - name: import snapshot and run Forest
         run: |
-          forest --chain calibnet --target-peer-count 50 --encrypt-keystore false --halt-after-import --import-snapshot /tmp/snapshots/*.car
+          forest --chain calibnet --target-peer-count 50 --encrypt-keystore false --halt-after-import --import-snapshot $SNAPSHOT_DIRECTORY/*.car
           forest-cli --chain calibnet db stats
           forest --chain calibnet --target-peer-count 50 --encrypt-keystore false --detach
       - name: wait for sync and check health
         run: |
           forest-cli sync wait
           forest-cli --chain calibnet db stats
-      - name: kill Forest process, print forest logs
+      - name: export snapshot and sync forest with exported snapshot
```

Comment:
It doesn't sync in this step, no? It imports the snapshot. If we want to sync, we need to `sync wait` like above.

Also, it may be a good exercise to check if the `shasum` of the exported snapshot is correct.

Reply:
We decided with @lemmih that just snapshot export would do. We would later add a snapshot verify command to validate exported snapshots.

Reply:
I've added the sha256 check step in CI